src/client/Engine/Caching.jsx

```diff
@@ -66,9 +66,9 @@ const Caching = ({ onChange, caching }) => (
           name="engine.caching.archive.retentionDuration"
           label="Retention Duration"
           value={caching.archive.retentionDuration}
-          defaultValue={8760}
+          defaultValue={720}
           valid={validation.engine.caching.archive.retentionDuration}
-          help={<div>How long to wait for the files to get delete in the archive folder (in hours)</div>}
+          help={<div>Retention period of archived files (in hours)</div>}
```

Comment:
Can you update the main 'help' section inside the OIbTitle ?
Please check that it's coherent with the new fields names and add the retentionDuration documentation (especially about the 0 value that means the files will be kept forever)